packages/app/src/containers/LifeStyle/index.tsx

```diff
@@ -0,0 +1,186 @@
+    vname: string;
+    vaditional: string;
+    vaditional1: string;
+}
+
+interface OwnProps {
+    navigation: any;
+    route: any;
+}
+
+interface InputAreaProps {
+    placeholder: string;
+    name: string;
+    value: any;
+    onChange: any;
+    searchIcon?: any;
+}
+
+type Props = OwnProps;
+
+const InputArea: React.FC<InputAreaProps> = ({ placeholder, name, value, onChange, searchIcon }) => {
+    return (
+        <View style={styles.inputView}>
+            <Input
+                containerStyle={[styles.containerStyle, { width: '100%' }]}
+                inputStyle={[styles.inputStyle]}
+                inputContainerStyle={[styles.inputContainerStyle, { marginTop: searchIcon ? -5 : 0 }]}
+                onChangeText={(value) => onChange(name, value)}
+                rightIcon={searchIcon}
+                value={value}
+                placeholder={searchIcon ? "Symptom" : placeholder} />
+        </View>
+    )
+}
+
+class LifeStylePage extends React.Component<Props, LifeStyleState> {
+    constructor(props: Props) {
+        super(props);
+
+        this.state = {
+            isVitamins: false,
+            isExercise: false,
+            isTeam: false,
+            vname: '',
+            vaditional: '',
+            vaditional1: '',
+        };
+    }
+
+    public componentDidMount = () => {
+    }
+
+
+    public render() {
+        const { isVitamins, isExercise, isTeam } = this.state;
+        return (
+            <React.Fragment>
+                <Header
+                    title={'PAIN IN THE APP'}
+                    leftIcon={true}
+                    rightIcon={true}
+                    navigation={this.props.navigation} />
+                <View style={styles.container}>
+                    <Text style={styles.heading}>LifeStyle</Text>
+                    <View style={styles.row}>
+                        <Text>Vitamins</Text>
+                        <View style={styles.switchrow}>
+                            <Text style={styles.yesNo}>{isVitamins ? "Yes" : "No"}</Text>
+                            <Switch
+                                trackColor={{ false: "#767577", true: "#035EC7" }}
+                                thumbColor={isVitamins ? "#035EC7" : "#f4f3f4"}
+                                ios_backgroundColor="#3e3e3e"
+                                onValueChange={this.toggleSwitchVitamins}
+                                value={isVitamins}
+                            />
+                        </View>
+                    </View>
+                    {
+                        isVitamins &&
+                        <View style={[styles.row, { alignItems: 'flex-start', marginTop: 20 }]}>
+                            <Text>Types</Text>
+                            <View style={{ width: '60%', marginBottom: 20 }}>
+                                <InputArea
+                                    placeholder="Vitamin Name"
+                                    name="vname"
```

Comment:
maybe try a less confusing name for the field, something simple like 'vitaminName' 1, 2, and 3

Reply:
ok @petermsouzajr